### PR TITLE
[TASK] Avoid Hungarian notation in a testcase

### PR DESCRIPTION
**Pull request by oliverklee**

### CODE REVIEW

File: tests/ParserTest.php [hunk 1]
```diff
@@ -1034,42 +1034,42 @@ public function lineNumbersParsing(): void
...
                 }
             }
         }
 
-        $aUrlExpected = [7, 26]; // expected line numbers
-        $aUrlActual = [];
-        foreach ($oDoc->getAllValues() as $oValue) {
-            if ($oValue instanceof URL) {
-                $aUrlActual[] = $oValue->getLineNo();
+        $expectedLineNumbers = [7, 26]; // expected line numbers
```
Comment (JakeQZ) [inline tests/ParserTest.php:1047]: We can lose the now-redundant comment.

File: tests/ParserTest.php [hunk 2]
```diff
@@ -1034,42 +1034,42 @@ public function lineNumbersParsing(): void
...
+        $actualUrls = [];
```
Comment (JakeQZ) [inline tests/ParserTest.php:1048]: This should be renamed `$actualLineNumbers`.

File: tests/ParserTest.php [hunk 3]
```diff
@@ -1034,42 +1034,42 @@ public function lineNumbersParsing(): void
...
+        foreach ($document->getAllValues() as $value) {
+            if ($value instanceof URL) {
+                $actualUrls[] = $value->getLineNo();
             }
         }
 
         // Checking for the multiline color rule lines 27-31
-        $aExpectedColorLines = [28, 29, 30];
-        $aDeclBlocks = $oDoc->getAllDeclarationBlocks();
+        $expectedColorLines = [28, 29, 30];
```
Comment (JakeQZ) [inline tests/ParserTest.php:1056]: I'd prefer `$expectedColorLineNumbers`.

File: tests/ParserTest.php [hunk 4]
```diff
@@ -1034,42 +1034,42 @@ public function lineNumbersParsing(): void
...
-        $aRules = $oDeclBlock->getRules();
+        $secondDeclarationBlock = $declarationBlocks[1];
+        $rules = $secondDeclarationBlock->getRules();
         // Choose the 2nd one
-        $oColor = $aRules[1]->getValue();
-        self::assertSame(27, $aRules[1]->getLineNo());
+        $valueOfSecondRule = $rules[1]->getValue();
+        self::assertSame(27, $rules[1]->getLineNo());
 
         $aActualColorLines = [];
```
Comment (JakeQZ) [inline tests/ParserTest.php:1065]: This Hungarian variable has been missed.

**Review by JakeQZ — CHANGES_REQUESTED** [review]
Good to introduce some new variables for the arrays that were temporarily assigned to a variable for storing a value from that array.

A couple of new names don't seem right or completely clear, and one rename has been missed.

**Comment by oliverklee** [comment]
Updated and repushed.

**Review by JakeQZ — APPROVED** [review]

**Pull request merged**